xml/favicons.xml

```diff
@@ -1962,6 +1962,13 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:mitre:caldera:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^e4ce127909d4697b97bf404a42e7c428$">
+    <description>mitmweb web interface for mitmproxy - https://github.com/mitmproxy/mitmproxy</description>
+    <example>e4ce127909d4697b97bf404a42e7c428</example>
+    <param pos="0" name="service.vendor" value="mitmproxy"/>
+    <param pos="0" name="service.product" value="mitmproxy"/>
```

Comment:
This, and below, should generate a CPE `a:mitmproxy:mitmproxy`

Reply:
Good catch! I recall looking at the CPE for this one so I'm not sure what happened to those changes.